### ISSUE TITLE
aria-sort - Is not identifiable by the assistive tools.

### ISSUE THREAD

**Issue by SukumarSV** [mdn/content#19093]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort

### What specific section or headline is this issue about?

aria-sort - Is not identifiable by the assistive tools.

### What information was incorrect, unhelpful, or incomplete?

Ive gone through the example of sortable table, the change occurred on sorting is not announced to the user. So the sorting order is not well define. We can introduce `ARIA-valuenow` kind of attribute to announce immediately. 

### What did you expect to see?

We can introduce `ARIA-valuenow` kind of attribute to announce immediately. 

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/accessibility/aria/attributes/aria-sort`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-sort
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/accessibility/aria/attributes/aria-sort/index.md
* Last commit: https://github.com/mdn/content/commit/6028b116c2ea52c9e5d32e8c15fa850e2cb8e3ac
* Document last modified: 2022-07-15T05:25:09.000Z

</details>

**Comment by bsmth** [comment]
Thanks for reporting! I think there's a misunderstanding: `aria-sort` is not updated automatically, it's the developer's job to set or toggle the attribute (via JavaScript). Screen readers use the attribute to describe the current sorting order.

Here's an example:
- https://www.w3.org/WAI/content-assets/wai-aria-practices/patterns/table/examples/js/sortable-table.js

From the ARIA guide:  

https://www.w3.org/WAI/ARIA/apg/patterns/table/examples/sortable-table/

Does that make sense? Going to close this for now.  Thanks a lot! :)
